tests/step-119.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+# Test:
+#   Test legacy transform introduced in PR #125 by updating from 49464f09f0 to HEAD
+#   - split global githooks.shared values : `legacy_transform_adjust_local_paths`
+#   - move local paths in .shared to local githooks.shared : `legacy_transform_split_global_shared_entries`
+
+cleanup() {
+    # Remove the whole install to
+    # not make external script which calls uninstall
+    # to dispatch into this uninstall!
+    git hooks uninstall --global >/dev/null 2>&1
+}
+
+trap 'cleanup' EXIT
+
+git clone https://github.com/rycus86/githooks.git /var/lib/githooks-original
+
+cd /var/lib/githooks-original &&
+    git checkout master &&
+    git checkout -b master-old &&
+    git checkout master &&
+    git reset --hard 49464f09f0 || exit 1
+
+# Install old version
+if ! sh /var/lib/githooks/install.sh --clone-url /var/lib/githooks-original --clone-branch "master"; then
```

Comment:
Thats, true, but since we have a dispatch which will stay there I think it is ok, But you are correct.
Here I take the current install.sh because I know it supports --clone-url --clone-branch and such...

Reply:
For this test I can actually use `/var/lib/githooks-original/install.sh` yes